lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -791,6 +797,49 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
     }
   }
 
+ void drawRangeAnnotation(Canvas canvas, Size viewSize) {
+    if (data.rangeAnnotations == null) {
+      print('is null');
+      return;
+    }
+
+    final Size chartUsableSize = getChartUsableDrawSize(viewSize);
+
+    if (data.rangeAnnotations.hasVerticalRangeAnnotations) {
+      for (VerticalRangeAnnotation anno in data.rangeAnnotations.verticalRangeAnnotations) {
```

Comment:
I don't prefer this kind of naming, it is annoying and it will make the code complex in the future.
please make a descriptive name, it's okay if it's too long, just pick a descriptive name.

Reply:
a5ecde7